server/views/pages/arrivingToday.njk

```diff
@@ -39,10 +39,10 @@
                 </thead>
                 <tbody class="govuk-table__body">
                     {% for prisoner in prisoners %}
-                        {% set prisonerName = prisoner.lastName | capitalize +', '+prisoner.firstName | capitalize %}
+                        {% set prisonerName = (prisoner.lastName +', '+prisoner.firstName) | nameCase %}
```

Comment:
See other comment... could implement and use `formatName` from the profile repo

Reply:
Have updated